src/video.tsx

```diff
@@ -253,6 +251,7 @@ const Video = React.forwardRef<VideoElement, VideoProps>(
                 aspectRatio={ratio}
                 width={props.width}
                 height={props.height}
+                dataAttributes={dataAttributes}
```

Comment:
set a default value for the component-name:

`dataAttributes={{'component-name': 'Video', ...dataAttributes}}`

Reply:
fixed